engine/src/flutter/lib/ui/semantics.dart

```diff
@@ -339,6 +339,29 @@ class SemanticsAction {
   String toString() => 'SemanticsAction.$name';
 }
 
+/// An enum to describe the role for a semantics node.
+///
+/// The roles are translated into native accessibility roles in each platform.
+enum SemanticsRole {
+  /// Does not represent any role.
+  none,
+
+  /// A tab button.
+  ///
+  /// see also:
+  ///  * [tabBar], which is the role for containers of tab buttons.
+  tab,
```

Comment:
Should this role be called tabButton to avoid any confusion on whether a `tab` is the button or the content of the tab?

Reply:
I think tab is a more appropriate name since it has the same meaning in flutter and all platforms. They all meant the button to switch tab panels.